CHANGELOG.md

```diff
@@ -2,48 +2,45 @@
-- Added `aswap` package. Atomic swap implementation is now separate from `escrow`.
-- Added `migration` package. Schema versioning can be implemented by relying on
-  functionality provided by this package.
-- `gconf` package was reimplemented from scratch
-- `x/cash` is using new `gconf` package for configuration. New genesis path is
-  used. To update genesis file, replace "gconf": { "cash:xyz": "foo" }
-  with "conf": { "cash": { "xyz": "foo" } }
-- Introducing go modules instead of dep
-- Removed support for go 1.10
-- Added support for go 1.12
-- Bumped minimum required version of go to 1.11.4+ as otherwise some commands
-  fail because of go mod constraints
+
+- Tendermint is upgraded to version 0.31.5
+- New `x/aswap` extension implementing atomic swap functionality. Atomic swap
+  implementation is separated from `x/escrow`
+- `x/cash` is using the new `gconf` package for configuration. New genesis path
+  is used. To update genesis file, replace `"gconf": { "cash:xyz": "foo" }` with
+  `"conf": { "cash": { "xyz": "foo" } }`
+- Removed support for Go 1.10. Minimal required version is not 1.11.4.
+- Added support for Go 1.12
 
 Breaking changes
 
-- x/paychan is using a wall clock timeout instead of block height
-- x/cash is using schema versioned model and messages
-- x/paychan is using schema versioned model and messages
-- x/escrow is using schema versioned model and messages
-- x/distribution is using schema versioned model and messages
-- x/namecoin is using schema versioned model and messages
-- x/sigs is using schema versioned model and messages
-- x/multisig is using schema versioned model and messages
-- x/msgfee is using schema versioned model and messages
-- x/currency is using schema versioned model and messages
-- x/nft is using schema versioned model and messages
-- x/validators is using schema versioned model and messages
+- Dependency management was migrated to Go modules. `dep` is no longer used or
+  supported.
+- `x/paychan` extension is using a wall clock for the timeout functionality
+  instead of relying on the block height
+- `gconf` package was reimplemented from scratch. Configuration can be changed
+  during the runtime using messages.
+
 
 ## 0.14.0
-- Tests were cleaned up and no use testify or convey packages. A new package
-  `weavetest/assert` contains test helpers
-- Simplify transaction message unpacking with weave.LoadMsg
-- Initial version of governance model
-- Crypto package improvements #495
-- Reduce timeout resolution in client/backend interface #447
-- Move timeout point to expired state (for atomic swaps) #446
-- Simplify validation logic in Handlers #437
-- Charge gas in x/sigs #429
-- New bump sequence Tx #388
-- Anti-spam fee changes    
+
+- Simplify transaction message unpacking with `weave.LoadMsg`
+- Initial version of the governance extension (`x/gov`)
+- Signature verification in `x/sigs` extension costs gas now
+- A new message `BumpSequenceMsg` for incrementing a user sequence value in
```

Comment:
Also missing:

* Adjustments to escrow processing (timeout inclusive)
* Added validator-subjective anti-spam fee

I do kind of like having a section "improvements" to mark some of the details there, like the overhaul of the testing framework. Even if it doesn't change the API it does document many days of work

Reply:
I understand that we want to show what we worked on because we are proud of the results. Is it really helpful for the readers though? Is it helpful for @webmaster128 (our only user that I know of :laughing:)